layouts/shortcodes/clients.html

```diff
@@ -12,15 +12,32 @@ <h2 class="font-bold text-2xl sm:text-4xl text-center mb-10">
       </span>
     </h2>
     {{ end }}
-    <ul class="flex flex-col md:flex-row md:flex-wrap items-center lg:justify-between gap-12 md:gap-8">
-      {{ range .clients_item}}
-      {{ if .show }}
-      <li>
-        <img loading="lazy" class="max-w-[180px]" src="{{ .image | absURL }}" alt="{{ .name }}">
-      </li>
-      {{ end }}
-      {{ end }}
-    </ul>
+    <div class="embla overflow-hidden" data-loop="true" data-scroll="1">
+      <div class="embla__viewport overflow-hidden">
+          <ul class="embla__container flex items-center touch-pinch-zoom">
+            {{ range .clients_item}}
+            {{ if .show }}
+            {{"<!-- Set flex basis to determine the number of slides/logos that will appear. -->" | safeHTML}}
+            <li class="embla__slide flex-[0_0_50%] md:flex-[0_0_33%] lg:flex-[0_0_20%]">
+              <figure class="flex justify-center items-center grayscale hover:grayscale-0">
+                <img class="w-28 h-auto max-h-14 object-contain" loading="lazy" src="{{ .image | absURL }}" alt="{{ .name }}" title="{{ .name }}">
+              </figure>
+            </li>
+            {{ end }}
+            {{ end }}
+          </ul>
+      </div>
+      <div class="flex justify-between lg:justify-center gap-x-8 items-center mt-4 p-2 sm:p-6">
```

Comment:
Displaying on the side of the logos, instead of below them, led to issues where the styles were blocked. I couldn't figure out how to prevent that, so they will appear below the logos for now.